packages/markers/src/browser/marker-manager.ts

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (C) 2017 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { injectable } from "inversify";
+import { Disposable, Event, Emitter } from "@theia/core/lib/common";
+import { JSONExt } from "@phosphor/coreutils";
+
+export interface Marker<T> {
+    id: string;
+    uri: string;
+    owner: string;
+    kind: string;
+    data: T;
+}
+
+export interface MarkerInfo {
+    counter: number;
+    uri: string
+}
+
```

Comment:
Please add comments on the marker and markerinfo fields , at least owner and counter are not obvious